app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,9 @@
+class CommentsController < ApplicationController
+
+	def create
+		params.permit!
```

Comment:
always sanitize the params explicitly.  (google Rails strong parameters)  unless you are absolutely certain it's not possible to craft a malicious request to setup data